src/Workspaces/VisualBasic/Portable/Indentation/VisualBasicIndentationService.vb

```diff
@@ -33,20 +33,21 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Formatting.Indentation
             Return _specializedIndentationRule
         End Function
 
-        Protected Overrides Function GetIndenter(syntaxFacts As ISyntaxFactsService,
+        Protected Overrides Function GetIndenter(document As Document,
```

Comment:
same as before. not sure why indenter abstraction is needed.

Reply:
i agree.  i can remove in a followup PR.  it's not clear at all to me why there was so much complexity here.  I think about 75% of all the code in the indentation codepaths was unnecessary, and PRs like this are trying to do something about that :)